packages/core/src/Provider/SharedWalletProvider/types.ts

```diff
@@ -0,0 +1,32 @@
+import { Ed25519KeyHashHex } from '@cardano-sdk/crypto';
+
+type ScriptType = number;
+
+export const MULTI_SIG_LABEL = 1854;
+
+interface MultiSigParticipant {
+  name: string;
+  description?: string;
+  icon?: string;
+}
+
+interface MultiSigParticipants {
+  [key: string]: MultiSigParticipant;
```

Comment:
Please find and use an appropriate type for this. Create a new one if it doesn't exist.
```suggestion
  [key: MoreSpecificType]: MultiSigParticipant;
```

Reply:
Fixed on cfcbae8